PERFTEST.PS1

```diff
@@ -8,8 +8,8 @@ Param(
     [parameter(Mandatory=$true)]  [String] $OutDir = "",
```

Comment:
validate params

Reply:
OutDir does not have to exist as we create Outdir with the UserName due to Linux home directory.